src/Symfony/Bundle/WebProfilerBundle/EventListener/WebDebugToolbarListener.php

```diff
@@ -138,6 +147,7 @@ public static function getSubscribedEvents()
     {
         return array(
             KernelEvents::RESPONSE => array('onKernelResponse', -128),
+            KernelEvents::REQUEST => array('onKernelRequest', 255),
```

Comment:
I'm not sure about the priority here, maybe this is not needed. It seems to work without and I don't know about any other listeners that could interfere here, but my knowledge is limited